ironic/drivers/modules/drac/management.py

```diff
@@ -183,10 +189,17 @@ def set_boot_device(node, device, persistent=False):
     :raises: DracOperationError on an error from python-dracclient.
     """
 
-    drac_job.validate_job_queue(node)
-
     client = drac_common.get_drac_client(node)
 
+    unfinished_jobs = drac_job.list_unfinished_jobs(node)
+    if unfinished_jobs:
```

Comment:
I think we need to change this logic a little:
if there are unfinished Bios jobs and the job queue cleaning step is about to run then delete just the unfinished bios jobs, otherwise, we want to validate the job queue as before.

Reply:
Done